vortex-ipc/src/messages.rs

```diff
@@ -186,15 +191,63 @@ impl<'a> WriteFlatBuffer for IPCArray<'a> {
             .collect_vec();
         let children = Some(fbb.create_vector(&children));
 
-        fba::Array::create(
+        let stats = collect_array_stats(fbb, self.1);
+
+        vfb::Array::create(
             fbb,
-            &fba::ArrayArgs {
+            &vfb::ArrayArgs {
```

Comment:
`fba`!

Reply:
This one is now from vortex::flatbuffers, which is why I went with vfb -- no strong preference though, I'll change it.